README.md

```diff
@@ -7,5 +7,5 @@ A crawler to find websites that exercise code in Firefox that is not covered by
 
 - Install requirements with `pip install -r requirements.txt`;
 - Install development requirements with `pip install -r test-requirements.txt`;
-- Run the `latest_cov_build.py` script to download the latest Firefox coverage build;
-- Run the `crawler.py` script.
+- Run the `coverage_crawler/latest_cov_build.py` script to download the latest Firefox coverage build;
```

Comment:
The scripts should actually be left in the top level directory. You can move parts (functions) from crawler.py into a new file under coverage_crawler/ and then import it in crawler.py